packages/uui-radio/lib/uui-radio-group.element.ts

```diff
@@ -49,22 +49,22 @@ export class UUIRadioGroupElement extends UUIFormControlMixin(LitElement, '') {
   }
   set value(newValue) {
     super.value = newValue;
-    if (!newValue || newValue === '') {
-      this._makeFirstEnabledFocusable();
-    }
-    this._updateRadioElementsCheckedState(newValue as string);
+    this.#updateRadioElementsCheckedState(newValue as string);
   }
 
-  private _selected: number | null = null;
+  #selected: number | null = null;
+  #radioElements: UUIRadioElement[] = [];
 
   constructor() {
     super();
-    this.addEventListener('keydown', this._onKeydown);
-    this.addEventListener('keypress', this._onKeypress);
+    this.addEventListener('keydown', this.#onKeydown);
+    this.addEventListener('keypress', this.#onKeypress);
```

Comment:
Do we really need `keypress` event since this has been deprecated for a while:
https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event

Reply:
No, we can remove that and let `keydown` stay.

Reply:
There's a few other places in code base the `keypress` event was used. I think these should be replaced by `keydown` or `keyup` events.